`fc_export()` bug fix

kenkomodo opened the pull request:
Fixes #32.

* Updated `fc_draw()` to output all current arguments into `params` to pass for use in `fc_export()`
* Updated `fc_export()` to use all current `fc_draw()` arguments passed via `params`

pasahe commented on R/fc_draw.R:
```diff
@@ -58,7 +58,7 @@ fc_draw.fc <- function(object, big.mark = "", box_corners = "round", arrow_angle
 
   #We have to return the parameters of the function in the attribute of object$fc
-  params <- c("arrow_angle", "arrow_length", "arrow_ends", "arrow_type")
+  params <- c("big.mark", "box_corners", "arrow_angle", "arrow_length", "arrow_ends", "arrow_type", "title", "title_x", "title_y", "title_color", "title_fs", "title_fface", "title_ffamily")
```
Don't forget to add the new parameter `canvas_bg`

pasahe replied:
Don't forget to add the new argument `canvas_bg`.

pasahe commented:
@kenkomodo great! It's true, I forgot to tell you to add the new arguments to params. Don't forget to add the new argument `canvas_bg` from the other PR. If you add it, I will merge it.

Thanks for noticing!

kenkomodo commented:
@pasahe If you merge #31 first, then I'll rebase this PR again and resolve the merge conflict that will show up in this PR once `canvas_bg` is merged into the `main` branch from the other PR.

pasahe commented:
I have merged PR #30. Now you can modify it and I will merge it.

By the way, feel free to change your role from contributor to author of the package if you are ok with it. You have made a lot of valuable changes that have made a difference to the package. Many many thanks again!!

kenkomodo commented:
@pasahe Thanks for merging that so quickly! I've rebased and resolved merge conflicts.

Also - thanks for offering to elevate me to the author list! I really appreciate it. I've added that change to the `DESCRIPTION` file onto this PR. Look forward to seeing other ways I might be able to help with the package in the future!
